tests/e2e/steps_consumer_misbehaviour.go

```diff
@@ -241,13 +241,15 @@ func stepsCauseConsumerMisbehaviour(consumerName string) []Step {
 						},
 					},
 				},
-				chainID(consumerName): ChainState{
```

Comment:
Removed since our model state checker requires the chain to still produce blocks.